### ISSUE TITLE
Consider: Alt Text Input in Preview Modal?

### ISSUE THREAD

**Issue by jpdevries** [Sterc/mediamanager#78]
Maybe this is not needed because I assume one could insert alt text like this:
![](http://j4p.us/0d3P2l2k3J0I/Screen%20Shot%202017-07-08%20at%205.53.29%20AM.png)

but maybe an input field for alt text should be below the Categories and Tags inputs?
![](http://j4p.us/3J460t0c2f2C/Screen%20Shot%202017-07-08%20at%205.52.56%20AM.png)
